src/chapter_1_examples/hello_you/src/lib.rs

```diff
@@ -1,13 +0,0 @@
-fn greet() -> String {
```

Comment:
I could not get this example to work and noticed that others were using a binary built from `src/main.rs`, so I removed this.